[32mINFO    [0m utils.logger_util:request_util.py:29 准备发起GET请求，URL: http://101.42.36.43:8888/auth/verify
[32mINFO    [0m utils.logger_util:request_util.py:30 请求信息：{'headers': {'user_token': 'eyJhbGciOiJIUzI1NiJ9.eyJpZGVudGl0eSI6IlVTRVIiLCJ1c2VySWQiOjE3LCJpYXQiOjE3NjA4NTUxMjksImV4cCI6MTc2MDk0MTUyOX0.tfStFYSaF8BGC_RyWSnyKrvdPy4AFhGpIeQD5_z4G9g'}}
[32mINFO    [0m utils.logger_util:request_util.py:34 响应状态码：200
[32mINFO    [0m utils.logger_util:request_util.py:35 响应内容：{"code":200,"data":{"valid":true,"identity":"USER","userId":17},"msg":"Token验证成功"}